packages/urlstate/next/useUrlState/README.md

```
@@ -56,3 +59,11 @@ Updates both the state and the URL.
 - `value?: T | Partial<T> | (currState: T) => T` - Optional new state value or a function that receives the current state and returns the new state.
 - `options?: Options` - Optional options object. When `replace` is true it will use router.replace. Nextjs `scroll` is `false` by default.
 
+## `reset`
+
+Updates both the state and the URL.
+
+### Parameters:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)